app/controllers/users_controller.rb

```diff
@@ -9,6 +9,7 @@ def destroy
 
   def destroy_user
     user = current_user || User.find_by(confirmation_token: session[:user_confirmation_token])
+    session.delete(:user_confirmation_token)
```

Comment:
@pkarman one issue with this is that the `:sp_session` info gets deleted as well. Since the user is abandoning sign up, does that seem reasonable?